src/glue/table.ts

```diff
@@ -46,20 +138,62 @@ export class Table extends Resource {
   private readonly _storageParameters: {[key: string]: string} = {};
 
   // Input properties
-  public readonly compressed?: boolean;
-  public readonly dataFormat?: DataFormat;
-  public readonly database: Database;
-  public readonly description?: string;
-  public readonly location?: string;
-  public readonly name?: string;
-  public readonly owner?: string;
-  public readonly retention?: Duration;
-  public readonly serdeName?: string;
-  public readonly tableType?: TableType;
-  public readonly targetTable?: Table;
-  public readonly viewExpandedText?: string;
-  public readonly viewOriginalText?: string;
-  public readonly storedAsSubDirectories?: boolean;
+  /**
```

Comment:
Indentation looks off.

Reply:
Tabs vs Spaces issue. Should be fixed now